dockerfiles/init/modules/haproxy/templates/haproxy.cfg.erb

```diff
@@ -35,6 +35,9 @@ backend tomcat_bk
 
 backend nginx_bk
     mode http
+    http-request replace-header Cookie (.*)session-access-key=([^;]*);(.*) \1\3
```

Comment:
So correct regexp is: `http-request replace-header Cookie (.*)session-access-key=([^;]*);?(.*) \1\3`